HucaresWF/src/Hucares.Server.Client/DetectedPlateClient.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Hucares.Server.Client.Models;
+using Newtonsoft.Json;
+
+namespace Hucares.Server.Client
+{
+    public class DetectedPlateClient : IDetectedPlateClient
+    {
+        
+        public Uri HostUri { get; private set; } = new Uri("http://localhost:50510");
+        private readonly HttpClientHelper httpHelper;
+
+        public DetectedPlateClient(HttpClientHelper httpHelper = null)
+        {
+            this.httpHelper = httpHelper ?? new HttpClientHelper();
+        }
+        
+        public async Task<IEnumerable<DetectedLicensePlate>> GetAllDetectedMissingPlates()
+        {
+            var uri = "api/dlp/all";
+            var fullUri = new Uri(HostUri, uri);
+
```

Comment:
one line variables should not be seperated by new line

Reply:
Divided and conquered